JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/GymDAO.java

```diff
@@ -0,0 +1,99 @@
+package com.laba.solvd.database.DAO;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class GymDAO {
+
+	public void createGym(Gym gym) {
+		try (Connection connection = (Connection) ConnectionManager.getSQLConnection().get();) {

```

Comment:
What is the purpose of casting Connection object here? I just wonder if it works without casting.